src/test/java/com/iota/iri/service/snapshot/impl/LocalSnapshotManagerImplTest.java

```diff
@@ -69,12 +79,15 @@ public void tearDown() {
     }
     
     @Test
-    public synchronized void takeLocalSnapshot() throws SnapshotException {
+    public synchronized void takeLocalSnapshot() throws Exception {
```

Comment:
It was like this before so not asking to change, but just wondering...
Do we know why this is synchronized?

Reply:
No idea, i guess if we run tests in parralel, modifying the state would break something? just a guess!